src/main/java/bio/terra/cli/utils/mount/MountController.java

```diff
@@ -96,9 +100,90 @@ public int mountResources(boolean disableCache, @Nullable Boolean readOnly) {
         .sum();
   }
 
+  public void mountResource(String resourceName, boolean disableCache, @Nullable Boolean readOnly) {
```

Comment:
This method is doing a lot of the same logic as `mountResources` above, can they be combined? Either having `mountResources` call `mountResource` for each resource, using a shared `mountResourceWorker` method to handle the implementation, or somethine else?

Reply:
Done!